### ISSUE TITLE
final grade

### ISSUE THREAD

**Issue by Tdrinker** [jadedjane/cs506-project#4]
* build, make, test: 7 / 10 - I appreciate you including how to run your notebooks. but you should have a requirements.txt or write down the necessary libraries to install. Also you didn't include a test section.
* description of data processing and feature engineering done: 10/10
* visualization of data, modeling analysis: 15/15
* modeling results, clear argument proving you achieved your results: 11/15 - I appreciate the detailed discussion of feature importance. However, you lack justification for why you choose to use lightGBM (and you didn't describe what this model is). Why use a gradient boosting model? how is that related to the nature of your dataset? You also didn't discussion training specifics of hyperparameters and how they affected your model. the performance report of the model also need to be strengthened.

43/50
